query/appservice/appservice_web_app_remote_debugging_disabled.sql

```diff
@@ -0,0 +1,19 @@
+select
+  -- Required Columns
+  app.id as resource,
+  case
+    when configuration -> 'properties' ->> 'remoteDebuggingEnabled' = 'false' then 'alarm'
```

Comment:
I think we should give `alarm` when   'remoteDebuggingEnabled' = 'true'.
Please check once.
https://docs.microsoft.com/en-us/azure/governance/policy/samples/hipaa-hitrust-9-2#ports-services-and-similar-applications-installed-on-a-computer-or-network-systems-which-are-not-specifically-required-for-business-functionality-are-disabled-or-removed

Reply:
made the changes according to the requirement